src/views/Main.js

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState, setErrorMessage } from 'react';
+import { fetchCountries } from '../services/countries';
+import CountryCard from '../components/CountryCard/CountryCard';
+import './Main.css';
+import Dropdown from '../components/Dropdown/Dropdown';
+
+
+export default function Main() {
+  const [countries, setCountries] = useState([]);
+  const [continent, setContinent] = useState('All');
+  const [loading, setLoading] = useState(true);
+  const options = ['All', 'Europe', 'North America', 'South America', 'Asia', 'Africa', 'Oceania', 'Antarctica'];
+
+  useEffect(() => {
+
+    const fetchData = async () => {
+      try {
+        const resp = await fetchCountries();
+        setCountries(resp);
+        setLoading(false);
+      } catch (e) {
+        setErrorMessage('We are experiencing technical difficulties. Please try reloading the page.');
+      }
```

Comment:
When placing a setErrorMessage, usually this will be setting the error in state so we will want to add 
const [errorMessage, setErrorMessage] = useState(''); Then on the return, we can add <p>{errorMessage}</p>